src/matter/interaction/InteractionMessages.ts

```diff
@@ -95,9 +95,9 @@ export const TlvAttributeReport = TlvObject({
 
 export const TlvDataReport = TlvObject({
     subscriptionId: TlvOptionalField(0, TlvUInt32),
-    values: TlvField(1, TlvArray(TlvAttributeReport)),
+    values: TlvOptionalField(1, TlvArray(TlvAttributeReport)), // Mandatory but chip tool do not send it
```

Comment:
I was wondering if we should not instead add support for a "default value" at the TLV level.
1. This is already used at a couple of places and we cannot represent it (on commands).
2. This will make keep the code simple for the specific bug addressed here.

TlvField(index: number, schema: TlvSchema<T>, default?: T)
When we decode the TLV object, is the field is missing, we use "default" value instead.

Should be an easy change

Reply:
Good idea. I would add a todo then here and do a pr in matter.js because we need it from there together with a release.